### DISCUSSION TITLE
ENSO and other variability in 116 and 118

### DISCUSSION THREAD

**Discussion by islasimpson** [NCAR/cesm_dev#31] [2. Model bias]
[Here](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.118/cvdp/) are some CVDP diagnostics of runs 116 and 118, compared to run 112 and a couple of segments of 104 from @phillips-ad.

(1)  116 and 118 are both improved in terms of the dry slot issue...

![Screenshot from 2024-12-19 07-33-25](https://github.com/user-attachments/assets/e36a2b30-3a2f-448e-93f5-ed12b81bf0a5)

(2) ENSO teleconnections to the North Pacific look a bit better in 116 than they were in 112.  118 looks a bit bad though.  Maybe too short to say anything for sure about this.

![Screenshot from 2024-12-19 07-35-33](https://github.com/user-attachments/assets/ce77d66b-d6fe-490d-87fe-a0e9914ec5ec)

(3) In the plot above there are also indications that the structure of ENSO is too narrow in the latitudinal direction, particularly in 118.  Not sure if this would be related to all the MLD issues @gustavo-marques?

(4) 116 and 118 both seem to be lacking in tropical Pacific SST variability as you can see in the SST standard deviation

![Screenshot from 2024-12-19 07-39-21](https://github.com/user-attachments/assets/717a1226-5b08-4d77-88f3-872e15223489)

and the bias in this standard deviation from obs

![Screenshot from 2024-12-19 07-41-32](https://github.com/user-attachments/assets/b0d337bc-8579-4730-9b63-e97cf98e4d92)

and in the Nino3.4 monthly variance

![Screenshot from 2024-12-19 07-39-50](https://github.com/user-attachments/assets/9525ba4a-7a4c-48af-ab20-6f0120aa99a1)

So, in terms of how 116 and 118 are doing, in tropical Pacific and related variability it's mixed.  Dry slot issue is definitely better, but a lack of ENSO variance and perhaps an ENSO structure that's too latitudinally narrow are both features to look out for moving forward.

@gustavo-marques, @fobryan3, @dlawrenncar, @adamrher, @PeterHjortLauritzen, @swrneale, @cecilehannay, @phillips-ad

**Comment by islasimpson** [comment]
@megandevlan

**Comment by megandevlan** [comment]
My hunch is that 118 especially is still a little too short, but the ENSO variances are _super_ low compared to other simulations (using years 2-end of runs):
![Screenshot 2024-12-19 at 11 12 46 AM](https://github.com/user-attachments/assets/2d7ee762-ce70-4cd2-b704-0efc116cc792). 

But pretty surprisingly, the westward extent of the strongest correlation between SST anomalies and nino 3.4 index is improving a little bit (at least for the 0.5 contour). That said, the negative correlations are really strongly reduced too, which seems less good. 
![Screenshot 2024-12-19 at 11 14 27 AM](https://github.com/user-attachments/assets/08fbbe19-d666-4873-a269-799d3e6fb22b)
![Screenshot 2024-12-19 at 11 15 27 AM](https://github.com/user-attachments/assets/4bee83fe-f31a-4485-9533-8c04fb1ffe45)
